tests/lib/Rule/Internal/RemoveInterfaceWithMethods/Fixture/class_with_removed_method_call.php.inc

```diff
@@ -0,0 +1,52 @@
+<?php /** @noinspection ALL */
+
+use Ibexa\Rector\Tests\Rule\Internal\RemoveInterfaceWithMethods\Fixture\SomeInterface;
+use Symfony\Component\Console\Command\Command;
+
+class SomeClass implements SomeInterface
+{
+    public function firstMethod(): void
+    {
+
+    }
+
+    public function configure()
+    {
+        $this->firstMethod()->firstMethod($this->someMethod())->thirdMethod();
+    }
+
+    public function someMethod(): array
+    {
+        return ['ezplatform:some_old:command_with_interface'];
+    }
+
+    public function thirdMethod(): void
+    {
+
+    }
+}
+
+?>
+-----
+<?php /** @noinspection ALL */
+
+use Ibexa\Rector\Tests\Rule\Internal\RemoveInterfaceWithMethods\Fixture\SomeInterface;
+use Symfony\Component\Console\Command\Command;
+
+class SomeClass
+{
+    public function firstMethod(): void
+    {
+
+    }
+    public function configure()
```

Comment:
Not sure though if formatting is relevant here:
```suggestion

    public function configure()
```

Reply:
It is, as rector either dosent care about it, or I didnt found a proper way to avoid it. At this moment, I leave proper formatting to CS fixer.